src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -189,7 +189,10 @@ protected function describeContainerDefinition(Definition $definition, array $op
         $output .= "\n".'- Abstract: '.($definition->isAbstract() ? 'yes' : 'no');
 
         if (method_exists($definition, 'isAutowired')) {
-            $output .= "\n".'- Autowired: '.($definition->isAutowired() ? 'yes' : 'no');
+            $output .= "\n".'- Autowire: ';
+            $output .= $definition->isAutowired() ? implode(', ', array_map(function ($method) {
```

Comment:
`__construct` should be removed from the output - and when only `__construct` is autowired, we should display a simple "yes"
same for other descriptors imho